Updating stage4 links

10leej opened the pull request:
This PR updates the stage 4 download links for the script.

10leej committed:
Updating stage4 links

asarubbo commented:
Hi, basically I updated the tarballs but I forgot to 'git push' the repositories. I suppose you found them by bruteforcing the date, anyway you didn't change the md5sum, so it didn't work in the practice. I have updated the repo right now.

10leej commented:
I'll be honest I didn't even think of the md5sum'a

Sent from my T-Mobile 5G Device
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Agostino Sarubbo ***@***.***>
Sent: Wednesday, April 5, 2023 5:14:06 AM
To: asarubbo/gentoo-stage4 ***@***.***>
Cc: josh 10leej.com ***@***.***>; Author ***@***.***>
Subject: Re: [asarubbo/gentoo-stage4] Updating stage4 links (PR #8)


Hi, basically I updated the tarballs but I forgot to 'git push' the repositories. I suppose you found them by bruteforcing the date, anyway you didn't change the md5sum, so it didn't work in the practice. I have updated the repo right now.

—
Reply to this email directly, view it on GitHub<https://github.com/asarubbo/gentoo-stage4/pull/8#issuecomment-1497171328>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AARHLHJXD22GIDPA274QAXDW7UZV5ANCNFSM6AAAAAAWKRTSOE>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
